<launch>
    <node pkg="my_controller" type="data_collector.py" name="data_collector" output="screen">
        <param name="video_filename" value="$(find my_controller)/data/driving_recording.avi" />
        <param name="csv_filename" value="$(find my_controller)/data/movement_commands.csv" />
        <param name="frame_width" value="640" />
        <param name="frame_height" value="480" />
        <param name="fps" value="30.0" />
    </node>
</launch>


